docs/content/doc/help/faq.en-us.md

```diff
@@ -292,15 +241,72 @@ By default, your LFS token will expire after 20 minutes. If you have a slow conn
 
 You may want to set this value to `60m` or `120m`.
 
-## How can I create users before starting Gitea
+If git reports the following when pushing, you likely didn't push through Gitea, see [SSH issues](#ssh-issues).
+```
+error: exit status 127, message: bash: git-lfs-authenticate: command not found (try: 5/6)
+```
 
-Gitea provides a sub-command `gitea migrate` to initialize the database, after which you can use the [admin CLI commands]({{< relref "doc/usage/command-line.en-us.md#admin" >}}) to add users like normal.
+### Gitea is running slow
 
-## How can I enable password reset
+The most common culprit for this is loading federated avatars.  
+This can be turned off by setting `ENABLE_FEDERATED_AVATAR` to `false` in your `app.ini`  
+Another option that may need to be changed is setting `DISABLE_GRAVATAR` to `true` in your `app.ini`
 
-There is no setting for password resets. It is enabled when a [mail service]({{< relref "doc/usage/email-setup.en-us.md" >}}) is configured, and disabled otherwise.
+### Issues after updating
+
+Please always check the release notes / [changelog](https://github.com/go-gitea/gitea/blob/main/CHANGELOG.md) for breaking changes; we regularly have breaking changes on minor releases!
```

Comment:
We really try not to have Breaking changes on minor releases... 

There are breaking changes on 1.x.0 releases - we very much try not to make breaking changes on 1.x.y releases.

The main suggestion though should be to always jump to the latest minor release - that is go from:

`1.13.3 -> 1.13.7 -> 1.14.5` NOT `1.13.3 -> 1.13.4 -> 1.13.5 -> ...  -> 1.14.0 -> 1.14.1 ... -> 1.14.5`